cmd/state-tray/main.go

```diff
@@ -84,7 +90,11 @@ func run() error {
 
 	systray.AddSeparator()
 	mAutoStart := systray.AddMenuItem(locale.Tl("tray_autostart", "Start on Login"), "")
-	if autostart.New().IsEnabled() {
+	enabled, err := autostart.New().IsEnabled()
```

Comment:
You're using an instance of autostart below on line 154, should those be using the same instance?

Reply:
Yes, we can use the same instance there. 👍